core/frontend/src/components/autopilot/EndpointCreationDialog.vue

```diff
@@ -167,6 +172,20 @@ export default Vue.extend({
     showDialog(state: boolean) {
       this.$emit('change', state)
     },
+    updateIp() {
+      switch (this.edited_endpoint.connection_type) {
+        case EndpointType.udpin:
+        case EndpointType.tcpin:
+          this.edited_endpoint.place = '0.0.0.0'
+          break
+        case EndpointType.udpout:
+        case EndpointType.tcpout:
+          this.edited_endpoint.place = this.user_ip_address
+          break
+        default:
+          this.edited_endpoint.place = '/dev/ttyAMA1' // Serial3
```

Comment:
before anyone asks. I picked Serial3 because it is better than the curent "0.0.0.0" for serial ports, and Serial1 is required for sbus (and I think something else that I forgot about).
Eventually we should make this a select box with the serial ports available